pymongoexplain/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.0.0'
+__version__ = '1.1.0dev0'
```

Comment:
Should be `'1.1.0.dev0'`. Check out https://www.python.org/dev/peps/pep-0440/.

Reply:
Actually the next release after 1.0.0 would be 1.0.1 so the dev release should be `'1.0.1.dev0'`.

Reply:
Remove the final period. It should be `1.0.1dev0`

Reply:
Nope, it has to be `.dev0`. Check out the pep.